patches/net/minecraft/nbt/NbtIo.java.patch

```diff
@@ -1,5 +1,48 @@
 --- a/net/minecraft/nbt/NbtIo.java
 +++ b/net/minecraft/nbt/NbtIo.java
+@@ -72,12 +_,19 @@
+     }
+ 
+     public static void writeCompressed(CompoundTag p_128945_, Path p_309705_) throws IOException {
++        // Neo: write to a temporary file and move it to the correct location instead
```

Comment:
Should we maybe create a helper for atomic moves. Something like `CommonHooks.atomicWrite(Path, Consumer<Path>)`?